### DISCUSSION TITLE
Tag for "In Person" Meeting

### DISCUSSION THREAD

**Discussion by aajcounty** [code4recovery/12-step-meeting-list#551] [Q&A] [answered]
Hi there!

We're wondering if anyone knows if/when/how to find the plug-in (and App) pre-created permanent tag for "in person" (for type of meeting.... for filtering in person meetings only)... is there something in the works?   As a work around, we're thinking that we need to create a custom "in person" tag in our plugin , but are worried with software updates that we will loose our tag customization......

Please share any & all info you know.

Thanks!

** and my apologies if this has been addressed -- I scoured the forum and couldn't find anything! **

**Comment by joshreisner** [comment, marked as answer]
Hi @aajcounty, our systems automatically tag meetings with `Online`, `In-Person`, `Active`, `Inactive` -- more info on how it decides is here: https://github.com/code4recovery/tsml-ui#how-are-metatypes-like-active-and-online-calculated

We don't have a way to manually type these. This is an intentional choice for a good user experience; we don't want the information supplied about a meeting not to match up with its designation.

**Reply by aajcounty** [reply]
Hey @joshreisner -- thank you for the speedy response! 

So I checked out the link you provided.  And I get it (intellectually) however I don't see this sorting option in the drop down here where our local meeting plugin lists (on our District Site). https://medfordareaaa.org/meetings/ 

The webmaster of our district page says that its not a function in the back end for "in person" (doesnt see a meta data location).

So how do we make it "naturally" appear (like you alluded to in the link above)?  Where in the plugin do we "activate" this?

We're soon going to be moving the schedules to love on our Central Office Website.... currently moving if off Weebly to WP.... to accommodate the plugin.  So just trying to do our due diligence before we do so!

Any advice on all fronts appreciated!

**Reply by joshreisner** [reply]
I think the district site may be using an old version of the plugin, or perhaps they have customized the the appearance of the list page. They should be showing up like the dropdown on the portland site does, that is the normal plugin behavior.

The links above are another story, those look like a customization. We don't have the bandwidth to advise on how to achieve that sort of look, but it's definitely doable. Good luck!

**Comment by aajcounty** [comment]
Also.....

I LOVE how Portland Intergroup custom coded their plugin -- see blue buttons & ease of Type dropdown
https://www.pdxaa.org/index.php/meetings/?tsml-day=6&tsml-attendance_option=in_person

Is this a custom code or is there some additional plugin/code thats open source that someone can share with me?

I will email then now but just checking with ya'll!
<img width="1013" alt="Screen Shot 2021-10-11 at 11 18 29 AM" src="https://user-images.githubusercontent.com/92276350/136836274-4e13aabd-1035-4dda-93fe-ce6db4d20f7f.png">
